src/List.tsx

```diff
@@ -315,15 +315,13 @@ class List<Value = string> extends React.Component<IProps<Value>> {
       // autoscrolling for containers with overflow
       if (height + 20 < this.listRef.current!.scrollHeight) {
         let scrollingSpeed = 0;
-        if (clientY - top < AUTOSCROLL_ACTIVE_OFFSET) {
+        if (clientY - top < autoscrollActiveOffset) {
```

Comment:
This comment is related to the original logic that this PR does not change. 

If I understand correctly the logic here, this is related to a scrollable container. There is the assumption that `listRef` is it. Assuming that the assumption is right, the math below should not be related to the size of that scrollable container? In particular should not the `autoscrollActiveOffset` be a ratio to the `height` of that scrollable container?